libs/vkd3d/resource.c

```diff
@@ -3481,35 +3482,48 @@ bool vkd3d_create_texture_view(struct d3d12_device *device, const struct vkd3d_t
     view_desc.subresourceRange.baseArrayLayer = desc->layer_idx;
     view_desc.subresourceRange.layerCount = desc->layer_count;
 
-    if (desc->miplevel_clamp != 0.0f)
-        FIXME_ONCE("Cannot handle MinResourceLOD clamp of %f correctly.\n", desc->miplevel_clamp);
-
-    /* This is not correct, but it's the best we can do with existing API.
-     * It should at least avoid a scenario where implicit LOD fetches from invalid levels.
-     * TODO: We will need an extension with vkCreateImageView pNext specifying minLODClamp.
-     * It will be trivial to add in RADV at least ... */
-    if (desc->miplevel_clamp >= 1.0f)
+    /* If we clamp out of bounds, then don't make a view
+     * and use a NULL descriptor to stay in-spec. */
+    if (desc->miplevel_clamp <= (float)(desc->miplevel_count - 1))
```

Comment:
Is miplevel_count correct? It needs to be baseMip + levelCount basically, and needs to handle VK_REMAINING_MIP_LEVELS if that's a thing.